src/linear_algebra/basic.lean

```diff
@@ -2624,6 +2624,10 @@ def general_linear_equiv : general_linear_group R M ≃* (M ≃ₗ[R] M) :=
   (general_linear_equiv R M f : M →ₗ[R] M) = f :=
 by {ext, refl}
 
+@[simp] lemma general_linear_equiv_to_fun (f : general_linear_group R M) :
+  ⇑(general_linear_equiv R M f) = ((f : M →ₗ[R] M) : M → M) :=
```

Comment:
Why not:
```suggestion
  ⇑(general_linear_equiv R M f) = (f : M → M) :=
```
This feels like a more natural lemma to me.

Reply:
I tried switching over just now, but it broke some simps in `number_theory/modular`.

Reply:
That's because we're missing the lemma:
```lean
@[simp] lemma to_linear_apply {n R : Type*} [fintype n] [decidable_eq n] [comm_ring R]
  (M : @general_linear_group n R ‹_› _ _) (v : n → R) :
  (@general_linear_group.to_linear n ‹_› ‹_› R _ M) v = matrix.mul_vec_lin M v :=
rfl
```